docs/json.md

```diff
@@ -804,27 +811,27 @@ manually, using only the [KSerializer] class.
 If tweaking the abstract methods `transformSerialize`/`transformDeserialize`/`selectDeserializer` is not enough, 
 then altering `serialize`/`deserialize` is a way to go.
 
-There are several tidbits on custom serializers with [Json].
+Here are some useful things about custom serializers with [Json]:
 
 * [Encoder] can be cast to [JsonEncoder], and [Decoder] to [JsonDecoder], if the current format is [Json].
 * `JsonDecoder` has the [decodeJsonElement][JsonDecoder.decodeJsonElement] method and `JsonEncoder` 
-  has the [encodeJsonElement][JsonEncoder.encodeJsonElement] method.
-  which basically retrieve/insert an element from/to a current position in the stream.
-* Both [`JsonDecoder`][JsonDecoder.json] and [`JsonEncoder`][JsonEncoder.json] have the `json` property 
+  has the [encodeJsonElement][JsonEncoder.encodeJsonElement] method,
+  which basically retrieve and insert an element from/to a current position in the stream.
```

Comment:
`retrieve/insert` was correct. 'retrieve' corresponds to 'decodeJsonElement' and 'from', 'insert' to 'encodeJsonElement' and 'to'

Reply:
Yeah, I see. But the use of `/` in place of conjunctions is generally a bad style because it's hard to parse for non-native speakers. 
Anyway, rewrote this in the clearest way possible :)